### PR TITLE
Improve bundle size and add webpack-bundle-analyzer

### PR DESCRIPTION
**Pull request by t3chguy**
For https://github.com/vector-im/element-web/issues/2498

Before 14.5 MB bundle.js
image

After 13.28 MB bundle.js
image

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -207,8 +207,10 @@ module.exports = (env, argv) => {
                 // Same goes for js/react-sdk - we don't need two copies.
                 "matrix-js-sdk": path.resolve(__dirname, "node_modules/matrix-js-sdk"),
                 "matrix-react-sdk": path.resolve(__dirname, "node_modules/matrix-react-sdk"),
-                // and sanitize-html
+                // and sanitize-html & matrix-events-sdk & matrix-widget-api
                 "sanitize-html": path.resolve(__dirname, "node_modules/sanitize-html"),
+                "matrix-events-sdk": path.resolve(__dirname, "node_modules/matrix-events-sdk"),
+                "matrix-widget-api": path.resolve(__dirname, "node_modules/matrix-js-sdk"),
```
Comment (andybalaam) [inline webpack.config.js:213]: Should the rhs of line 213 be `matrix-widget-api`? It's `matrix-js-sdk` here.

**Review by andybalaam — CHANGES_REQUESTED** [review]
One question, looks good otherwise.

File: webpack.config.js (hunk 1 above)
Comment (t3chguy) [reply webpack.config.js:213]: Whoops yes, good spot

Comment (t3chguy) [reply webpack.config.js:213]:
```suggestion
                "matrix-widget-api": path.resolve(__dirname, "node_modules/matrix-widget-api"),
```

**Review by andybalaam — APPROVED** [review]

**Pull request merged**